Lib/fontTools/ttLib/tables/TupleVariation.py

```diff
@@ -599,12 +600,11 @@ def decompileSharedTuples(axisTags, sharedTupleCount, data, offset):
 
 
 def compileSharedTuples(axisTags, variations):
-	coordCount = {}
+	coordCount = defaultdict(int)
```

Comment:
there's also `collections.Counter`, not sure if it's any faster https://docs.python.org/3/library/collections.html#counter-objects

Reply:
Thanks. Used it in one place.